### ISSUE TITLE
Safari WebGL Shader error "'round' : no matching overloaded function found"

### ISSUE THREAD

**Issue by MaxTwentythree** [protectwise/troika#108]
Hi,
in Safari troika text isn't rendered anymore .. I receive the following WebGL Shader error:

```
"THREE.WebGLShader: gl.getShaderInfoLog() fragment
ERROR: 0:385: 'round' : no matching overloaded function found
```

It's this line in the Shader:

`385:   float ch = round(vTroikaTextureChannel);`

Unfortunatelly I'm not sure with which troika-three-text version this started .. 
I'm currently on 0.39.1
My three version is 0.125.2

Is this just me, or does someone else also have this problem?

**Comment by lojjic** [comment]
Oof, guess I need to be better about testing Safari in my release process. I'd been getting lazy.

I'll put out a fix shortly.

**Comment by lojjic** [comment]
I've released 0.39.2 with the fix, let me know if you still see issues.

**Comment by MaxTwentythree** [comment]
Yay, this was fast!
Thank you, it's working now :)
